Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
+		return "newspaperArticle";
```

Comment:
nit: too many indents

Zotero will not let you import items without a title, but also, in the absence of a title element on the page, the code in `scrape` will throw. You should make sure here that that element exists.